src/lib/Functions/handlePaste.ts

```diff
@@ -4,6 +4,18 @@ import { ClipboardEvent } from '../Model/domEventsTypes';
 import { getActiveSelectedRange } from './getActiveSelectedRange';
 import { pasteData } from './pasteData';
 
+const decimalSeparator = (1.1).toLocaleString().substring(1, 2);
+const thousandSeparator = (1000).toLocaleString().substring(1, 2);
+
+function parseFloatLocale(text: string): number {
+    return parseFloat(
+      text
+        .replace(thousandSeparator, "")
+        .replace(decimalSeparator, ".")
+        .replace(/[^0-9.-]/g, "")
+    );
+}
```

Comment:
I think it can be simplified to just:
```suggestion
const localeParseFloat(text: string): number  => parseFloat(text.replaceAll(".","").replaceAll(",","."));
```

Reply:
that _only_ works in places with "," as decimal separator. if you paste "100.5" (which should be one hundred and a half), it will turn to "1005" with your suggestion. Or if you paste "1,000" (one thousand), it will turn into 1.
Hence you need to first determine which symbol is used for which purpose.

Reply:
Ah, you're right - this won't work if the number is already in US format...